swagger.yaml

```diff
@@ -214,6 +214,83 @@ paths:
             application/json;charset=utf-8:
               schema:
                 $ref: "#/components/schemas/InternalServerError"
+  "/metaforms/{metaformId}/auditLogEntries":
+    get:
+      operationId: findAuditLogEntries
+      summary: Finds all audit log entries for metaform
+      description: Finds all audit log entries for metaform
+      tags:
+        - AuditLogEntries
+      parameters:
+        - name: metaformId
+          required: true
+          in: path
+          description: Metaform id
+          schema:
+            type: string
+            format: uuid
+        - name: userId
+          in: query
+          description: Filter results by user id. If this parameter is not specified all
+            replies are returned, this requires logged user to have proper
+            permission to do so
+          schema:
+            type: string
+            format: uuid
+        - name: createdBefore
+          description: Filter results created before specified time
+          in: query
+          schema:
+            type: string
+        - name: createdAfter
+          description: Filter results created after specified time
+          in: query
+          schema:
+            type: string
+        - name: modifiedBefore
+          description: Filter results modified before specified time
+          in: query
+          schema:
+            type: string
+        - name: modifiedAfter
+          description: Filter results modified after specified time
+          in: query
+          schema:
+            type: string
```

Comment:
Log entries are not changed after they are created, so wouldn't filters timeBefore and timeAfter do the job?